packages/juggler/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.tgz
+lib*/
+package
```

Comment:
Do we need a package-specific `.gitignore`? How does this differ from the top-level `.gitignore`?